### ISSUE TITLE
Differentiate style of headers from clickable items in dropdown menu

### ISSUE THREAD

**Issue by ebarry** [publiclab/plots2#9935]
## desired behavior

The Operations Team at Public Lab nonprofit requests a review towards a possible update of headers within the dropdown menus, specifically the Tools menu. 

**Additional context (optional)**
See the yellow highlight on Kits and Software? These are headers:  

![image-1](https://user-images.githubusercontent.com/161439/126509070-0b2e805b-f5f7-4b13-aecc-f077cbc38e17.png)

Could these headers be more unique from clickable links so that the headers so not appear to be broken links? 

Thanks!

**Comment by 2201monesh** [comment]
Hey I would like to work on this issue, can I take it?

**Comment by ebarry** [comment]
Great, thanks for writing back! Could you try out some different ways to emphasize the headings in this menu and respond with screenshots here in this issue?

**Comment by 2201monesh** [comment]
@ebarry sure can you tell me in which file its there.

**Comment by Tlazypanda** [comment]
Hey @2201monesh you can find it here - 
`app/views/layouts/_header.html.erb`

Lines 71 - 78
```
              <h6 class="dropdown-header">Kits</h6>
              <%= create_nav_dropdown_item("kits","layout._header.tools.kits_initiative") %>
              <%= create_nav_dropdown_item("//store.publiclab.org","layout._header.tools.store") %>
              <h6 class="dropdown-header">Software</h6>
              <%= create_nav_dropdown_item("https://mapknitter.org","layout._header.tools.map_knitter") %>
              <%= create_nav_dropdown_item("https://infragram.org","layout._header.tools.infragram") %>
              <%= create_nav_dropdown_item("https://spectralworkbench.org","layout._header.tools.spectral_workbench") %>
              <%= create_nav_dropdown_item("https://code.publiclab.org#r=all","layout._header.tools.code_community") %>
          
```

As @ebarry suggested the Kits and Software dropdown-headers styles can be experimented around a bit so that they can be easily distinguished from the other nav elements here. Hope this can help you get started :tada:

**Comment by 2201monesh** [comment]
@Tlazypanda Thanks for the guidance. Working on it

**Comment by ebarry** [comment]
Hi @2201monesh , have you run into any problems on this? Would love to hear from you either way 😄

**Comment by 2201monesh** [comment]
@ebarry thanks for your patience but I am really stuck can't solve it.

**Comment by jywarren** [comment]
Hi! For background, we use the Bootstrap 4 UI library's dropdown styles:

https://getbootstrap.com/docs/4.4/components/dropdowns/#headers

For comparison those are to the left below, while the next image is our website: 

![image](https://user-images.githubusercontent.com/24359/128052659-5d1a0c91-79dd-462c-903c-84b94ed2f37e.png)![image](https://user-images.githubusercontent.com/24359/128052755-cc0feea7-89de-4697-aff4-f9effdda6bee.png)

I've sometimes seen that headers are ALL CAPS and **BOLD** as well as smaller. 

We are seeing that the "Junction" font is overriding Bootstrap's default CSS:

https://github.com/publiclab/plots2/blob/a54bf46102c1951734ab94966b4f9e867b0b2762/app/assets/stylesheets/style.css#L41-L43

We could limit that https://github.com/publiclab/plots2/blob/a54bf46102c1951734ab94966b4f9e867b0b2762/app/assets/stylesheets/style.css#L41-L43

Here's a suggestion -- we add a new more specific style to implement extra style adjustments:

```css
h6.dropdown-header {
    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
    font-weight: bold;
    text-transform: uppercase;
    font-size: 75%;
    color: #aaa;
}
```

How does this look to you, @ebarry ?

![image](https://user-images.githubusercontent.com/24359/128053682-d9b97d2c-4e52-42d2-93fe-3d086c37b44a.png)

**Comment by ebarry** [comment]
Yes! Jess and I are approving. thanks, let's go with this.

**Comment by jywarren** [comment]
Great, made a FTO: https://github.com/publiclab/plots2/issues/9986

**Comment by noi5e** [comment]
The fix for this seems to be merged in #9986, so closing it.
